chutzpah/Samples/readme.txt

```diff
@@ -1,26 +0,0 @@
-Chutzpah Samples 
```

Comment:
I think we should keep this file. It shows some of the useful aspects of QUnit and Chutzpah